app/sections/about/about.component.html

```diff
@@ -4,12 +4,12 @@
   </h1>
 
   <div id="about-help-nav">
-    <a *ngFor="#menuItem of submenuItems"
-       [routerLink]="['AboutSubsection', {slug: menuItem.slug }]">
-      <span>
-        {{ menuItem.title }}
-      </span>
-    </a>
+    <!--<a *ngFor="#menuItem of submenuItems"-->
```

Comment:
No, but I haven't change this section to dynamic - so it'll be removed / changed anyway in the next commit